src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx

```diff
@@ -46,20 +51,20 @@ class TableBodyRow extends Component {
   };
 
   render() {
-    const { classes, rowData, columns, renderTd, expandLabel, trProps } = this.props;
+    const { classes, rowData, columns, renderTd, expandLabel, trProps, borderLessRows } = this.props;
     const { collapsed } = this.state;
     const mobileSortedColumns = columns.filter(col => col.responsiveProps.flexOrder > 0).sort(compareResponsiveOrder);
-    const spacers = findSpacers(mobileSortedColumns);
+    const { spacers, colsWithRows } = findSpacers(mobileSortedColumns);
     return (
       <Tr className={cn({ [classes.expanded]: !collapsed }, classes.row)} {...trProps}>
-        {columns.map(col => (
+        {colsWithRows.map(col => (
           <Td
             className={cn({
               [classes.firstMobileRow]: col.firstMobileRow,
             })}
             key={col.key || col.baseKey || col.padding}
             align={col.align}
-            borderBottom
+            borderBottom={collapsed || borderLessRows.indexOf(col.rowNr) === -1}
```

Comment:
seems like `indexOf` could be substituted with [`includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes).

Reply:
fixed